devito/operator/operator.py

```diff
@@ -376,8 +376,8 @@ def _lower_clusters(cls, expressions, profiler=None, **kwargs):
         # Build a sequence of Clusters from a sequence of Eqs
         clusters = clusterize(expressions, **kwargs)
 
-        # Lift iteration spaces surrounding PETSc equations to produce
-        # distinct iteration loops.
+        # Lift iteration space surrounding each PETSc solve to create
+        # distinct iteration loops
         clusters = petsc_lift(clusters)
```

Comment:
nitpicking, and OK to address this another day: instead of petsc_lift, I'd use a more generic name such as `petsc_preprocess`, which internally calls lift() and, in the future, who knows what other routines